framework/db/QueryBuilder.php

```diff
@@ -775,12 +775,14 @@ public function buildSelect($columns, &$params, $distinct = false, $selectOption
     }
 
     /**
-     * @param array $tables
+     * @param Query $query the [[Query]] object from which the SQL statement will be generated.
      * @param array $params the binding parameters to be populated
      * @return string the FROM clause built from [[Query::$from]].
      */
-    public function buildFrom($tables, &$params)
+    public function buildFrom($query, &$params)
```

Comment:
Fixed backwards compatibility, but travis-ci through this error:

> The command "tests/data/travis/apc-setup.sh" failed and exited with 1 during .

Has this relation with my commit? All test related with mysql hint index are passed on my local machine.